api/openapi-spec/swagger.json

```diff
@@ -19323,7 +19323,7 @@
   },
   "info": {
     "title": "Kubernetes",
-    "version": "v1.15.0"
```

Comment:
Check with Xiaoning. We might be able to use 0.2, but should not stay as Kubernetes

Reply:
It is auto genernated.